tray.c

```diff
@@ -264,6 +264,13 @@ OnNotifyTray(LPARAM lParam)
             }
         }
         break;
+
+    case WM_OVPN_RESCAN:
+        /* Rescan config folders and recreate popup menus */
+        DestroyPopupMenus();
```

Comment:
We already have those 3 functions called in 2 places (WM_RBUTTONUP and WM_LBUTTONDBLCLK) and now we add one more. Can we factor them out into RescanAndRecreate() function?

Reply:
Good point. Will force push with this change as well as fix for the tab vs space error above.